packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/en.ftl

```diff
@@ -5,3 +5,9 @@ create-new-password-header = Create new password
 account-restored-success-message = You have successfully restored your account using your account recovery key. Create a new password to secure your data, and store it in a safe location.
 # Feedback displayed in alert bar when password reset is successful
 account-recovery-reset-password-success-alert = Password set
+# An error case was hit that we cannot account for.
+account-recovery-reset-password-unexpected-error = Unexpected Error Encountered
+account-recovery-reset-password-email-resent = Email resent. Add accounts@firefox.com to your contacts to ensure a smooth delivery.
```

Comment:
```suggestion
account-recovery-reset-password-email-resent = Email resent. Add { $accountsEmail } to your contacts to ensure a smooth delivery.
```
Instead of including the accounts@firefox.com email address as part of the string, could you use a variable instead?

There are multiple reasons this would be preferable: 
* It's possible that translators may mistakenly translate or edit the email address
* There's no way for us to detect if/when the email address is input incorrectly for localized text
* The string can continue to be used without retranslation even if the email address needs to be updated in the future

Please also include a comment explaining the variable.
Below example for reference:
https://github.com/mozilla/fxa/blob/7f6039b08d61dbe31d5532fc2743c2624eb98ea2/packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/en.ftl#L11-L12

Reply:
Parameterized and comments added.